src/Writers/HTMLWriter.jl

```diff
@@ -1646,7 +1646,11 @@ function mdconvert(d::Dict{MIME,Any}, parent; kwargs...)
     elseif haskey(d, MIME"image/jpeg"())
         out = Documents.RawHTML(string("<img src=\"data:image/jpeg;base64,", d[MIME"image/jpeg"()], "\" />"))
     elseif haskey(d, MIME"text/latex"())
-        out = Utilities.mdparse(d[MIME"text/latex"()]; mode = :single)
```

Comment:
As far as I can tell, this `mdparse` call for LaTeX was introduced in #938. @fredrikekre I know this was ages ago, but do you remember whether there was any particular reason for this? I don't think we should expect `text/latex` representations be formatted as Markdown?

Reply:
Probably just a copy-paste error.